src/graphrag_toolkit/retrieval/retrievers/__init__.py

```
@@ -3,7 +3,8 @@
 
 from .chunk_based_search import ChunkBasedSearch
 from .entity_based_search import EntityBasedSearch
-from .traversal_based_retriever import TraversalBasedRetriever, WeightedTraversalBasedRetrieverType
+from .topic_based_search import TopicBasedSearch
```

Ruff (0.8.2)

`.topic_based_search.TopicBasedSearch` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)